packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1934,7 +1934,11 @@ export class LitNodeClientNodeJs
         for (const field of requiredFields) {
           const key: keyof BlsResponseData = field as keyof BlsResponseData;
 
-          if (!data[key] || data[key] === '') {
+          if (
+            data[key] === undefined ||
+            data[key] === null ||
```

Comment:
nit: i think `data[key] == null` will be true for both `null` and `undefined`

Reply:
![Screenshot from 2024-10-09 19-48-23](https://github.com/user-attachments/assets/43cfba8d-9f40-4d87-9356-8923bfd1eefc)
Sadly for us now, no. Javascript magic 🪄